packages/webview_flutter/android/src/main/java/io/flutter/plugins/webviewflutter/FlutterWebView.java

```diff
@@ -128,6 +131,12 @@ private void goForward(MethodCall methodCall, Result result) {
   private void reload(MethodCall methodCall, Result result) {
     webView.reload();
     result.success(null);
+    webView.stopLoading();
```

Comment:
This looks like it might need to be removed.

Reply:
😳️good catch. removed it.